EcoSoapBank/Model/ImpactStats.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  ImpactStats.swift
+//  EcoSoapBank
+//
+//  Created by Shawn Gee on 8/11/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import UIKit
+
+struct ImpactStats: Decodable {
+    
+    let stats: [Statistic: Int]
+
+    enum Statistic: String, CodingKey, CaseIterable {
+        case soapRecycled
+        case linensRecycled
+        case bottlesRecycled
+        case paperRecycled
+        case peopleServed
+        case womenEmployed
+        
+        var description: String {
+            switch self {
+            case .soapRecycled:
+                return "soap recycled"
+            case .linensRecycled:
+                return "linens recycled"
+            case .bottlesRecycled:
+                return "bottle amenities/nrecycled"
+            case .paperRecycled:
+                return "paper recycled"
+            case .peopleServed:
+                return "people served"
+            case .womenEmployed:
+                return "women employed"
+            }
+        }
+        
+        var image: UIImage {
+            switch self {
+            case .soapRecycled:
+                return UIImage(named: "Bottles")!
+            case .linensRecycled:
+                return UIImage(named: "Bottles")!
+            case .bottlesRecycled:
+                return UIImage(named: "Bottles")!
+            case .paperRecycled:
+                return UIImage(named: "Bottles")!
+            case .peopleServed:
+                return UIImage(named: "Bottles")!
+            case .womenEmployed:
+                return UIImage(named: "Bottles")!
+            }
+        }
+    }
```

Comment:
It could be worth putting description, image, and possibly even amountString into extensions to clean things up a tad, but that's definitely not high priority.

Reply:
I would find that to be a bit of an over-use of the extensions-as-code-organization scheme, but that's just my personal preference.